src/main/java/org/folio/service/fieldname/FieldNameConfig.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.service.fieldname;
+
+import org.folio.service.fieldname.loader.IdentifierTypesLoader;
+import org.folio.service.fieldname.loader.ReferenceDataLoader;
+
+import java.util.Map;
+
+public enum FieldNameConfig {
+
+  IDENTIFIERS("identifiers", "$.{recordType}[*].identifiers[?(@identifierTypeId=={id})].value", new IdentifierTypesLoader()),
+  FIXED_LENGTH_DATA_ELEMENT("metadata.createdDate", "$.{recordType}.metadata.createdDate", MetadataParametersConstants.getFixedLengthDataElement()),
+  HR_ID("hrid", "$.{recordType}.hrid");
+
+  private final String id;
```

Comment:
let's call it fieldId or something let's reserved id only for referring to UUIDs

Reply:
sure, thanks